libs/agno/agno/workflow/workflow.py

```diff
@@ -89,7 +89,7 @@ def __init__(
     ):
         self.name = name or self.__class__.__name__
         self.workflow_id = workflow_id
-        self.description = description
+        self.description = description or self.__class__.description
```

Comment:
We should probably use a proper docstring parser here? Although this is likely good enough for now.